lib/Analysis/AxisInfo.cpp

```diff
@@ -209,6 +209,33 @@ ChangeResult AxisInfoAnalysis::visitOperation(
     }
     curr = AxisInfo(contiguity, divisibility, constancy);
   }
+
+  // CmpI
+  if ((llvm::dyn_cast<arith::CmpIOp>(op) ||
+       llvm::dyn_cast<triton::gpu::CmpIOp>(op)) &&
```

Comment:
why do we need to deal with arith::CmpIOp here? It is not supposed to carry any tensor types in Triton's context.

Reply:
resolved, ignore

Reply:
The `arith::CmpIOp` is also used in Triton dialect phase, to represent tensor computation from Frontend.